dts/bindings/mipi-dbi/renesas,smartbond-mipi-dbi.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+include: [mipi-dbi-controller.yaml, pinctrl-device.yaml]
+
+description: Renesas Smartbond(tm) MIPI DBI Host
+
+compatible: "renesas,smartbond-mipi-dbi"
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  reset-gpios:
+    type: phandle-array
+    description: |
+      Reset GPIO pin. Used to reset the display during initialization.
+
+  te-enable:
+    type: boolean
+    description: |
+      Boolean to indicate whether the tearing effect (TE) signal is available or not.
+
+  te-inversion:
+    type: boolean
+    description: |
+      Boolean to apply an inversion on the TE signal that triggers the MIPI DBI controller.
+
+  dma-prefetch:
+    type: int
+    enum:
+      - 0 # Prefetch mechanism is disabled
```

Comment:
You can use an enum type here of `string` and then use `DT_INST_ENUM_IDX` in the code to get the value you need, this will be easier for the user to follow